Client/Frontend/Browser/TabManager.swift

```diff
@@ -55,20 +55,17 @@ class TabManager : NSObject {
     private let defaultNewTabRequest: NSURLRequest
     private let navDelegate: TabManagerNavDelegate
     private var configuration: WKWebViewConfiguration
-    let storage: RemoteClientsAndTabs?
+    let profile: Profile
```

Comment:
I'm very slightly concerned that this will cause a retain cycle, so maybe `unowned` here?